.github/workflows/ci-nix.yml

```diff
@@ -126,6 +126,15 @@ jobs:
         if: (github.event_name != 'pull_request' || matrix.build-in-pr) && matrix.run-tests && (matrix.host != 'macos')
         run: nix build -L .#wasm32-unknown.ci.wasmTest --keep-failed
 
+
+      - name: Cross Tests
+        if: (github.event_name != 'pull_request' || matrix.build-in-pr) && matrix.run-tests
+        run: |
+          # the default tmp dir is too long (/home/ubuntu/actions-runner/_work/_temp/)
+          mkdir /home/ubuntu/tmp
```

Comment:
Ubuntu? :S

Any reason just `/tmp` doesn't work? Then at least `$HOME/tmp` maybe so I can pretend we are not running Ubuntu? :D

Reply:
Running on ubunutu in buildjet gives us user:group as ubuntu:ubuntu. I was hoping to use /tmp, but that's owned by root. I updated to $HOME/tmp.

Reply:
```
> ls -alh / | grep tmp
drwxrwxrwt  73 root root  44K Jan  9 08:46 tmp
```

`/tmp` is owned by `root` on every Linux system. It uses sticky bit `t` to allow creation of entries by other users. There are probably some other reasons wny action runner would default to something else, probably invalid ones. :D